DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -10,6 +10,7 @@
 #include "DataFormats/VertexReco/interface/Vertex.h"
 #include "DataFormats/VertexReco/interface/VertexFwd.h"
 #include "FWCore/Utilities/interface/thread_safety_macros.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
this seems unnecessary
but if it is, follow #32859 to remove dependence on FWCore/MessageLogger in the header file here
```suggestion
```